index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <div class="Headerback">
+    <header class="header">
+        <a class="logo" href="">My Logo</a>
+        <nav>
+            <img class="onion" src="image/Union.png" alt="" />
+        </nav>
+    </header>
+    </div>
+
+
+    <div class="container">
+
+        <section class="backgroundimg">
```

Comment:
### OPTIONAL


- [ ] Kindly add the right background image, this is currently missing in your design 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_389](https://user-images.githubusercontent.com/69638013/185757296-66163ca8-6038-454e-86bb-75250988352a.png)  |  ![Screenshot_392](https://user-images.githubusercontent.com/69638013/185757306-b49ad07a-233f-4e14-924f-25c2ccc74dfd.png)